index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio setup and mobile version skeleton</title>
+    <title>yayner's portfolio home page</title>
```

Comment:
- [ ] As requested by the previous code reviewer, please kindly replace you `page title` to something like `Yayner's Portfolio or Yayner Software Developer` since it is a single page application with various sections, appending home page to the title, will present as if the content is only for the home page. Please kindly update your page title.